<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA</class>
 <widget class="QWidget" name="RSA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic;&quot;&gt;RSA Cipher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="plaintext_edit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Cantarell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ciphertext_edit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Cantarell&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>加密-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>266</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>解密&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>350</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>私钥a:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>私钥b：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
